pymeasure/adapters/adapter.py

```diff
@@ -59,16 +64,24 @@ def read(self):
         """
         raise NameError("Adapter (sub)class has not implemented reading")
 
-    def values(self, command, separator=',', cast=float):
+    def values(self, command, separator=',', cast=float, preprocess_reply=None):
         """ Writes a command to the instrument and returns a list of formatted
         values from the result 
 
         :param command: SCPI command to be sent to the instrument
         :param separator: A separator character to split the string into a list
         :param cast: A type to cast the result
+        :param preprocess_reply: optional callable used to preprocess values
+        received from the instrument. The callable returns the processed string.
+        If not specified, the Adapter default is used if available, otherwise no
+        preprocessing is done.
```

Comment:
```suggestion
        :param preprocess_reply: optional callable used to preprocess values
            received from the instrument. The callable returns the processed string.
            If not specified, the Adapter default is used if available, otherwise no
            preprocessing is done.
```